src/Plugin/GraphQL/DataProducer/Routing/RouteLoad.php

```diff
@@ -24,6 +24,10 @@
  *   consumes = {
  *     "path" = @ContextDefinition("string",
  *       label = @Translation("Path")
+ *     ),
+ *     "language" = @ContextDefinition("string",
+ *       label = @Translation("Language"),
+ *       required = FALSE
```

Comment:
we should specify a default value `und` here, which is Language::LANGCODE_NOT_SPECIFIED